nightingale_perception/README.md

```diff
@@ -0,0 +1,9 @@
+# Perception
+launch point cloud body filtering and point cloud to laser scan conversion with <roslaunch nightingale_perception pt_cloud_process.launch>
+body filtered points topic is /camera/depth/color/points_body_filtered
+point cloud converted to laser scan topic is /movo/camera_scan
+
+# Map Frame Visualizer
```

Comment:
did u mean to push this? map frame viz is not in nightingale_perception

Reply:
removed